build-system/server.js

```diff
@@ -19,7 +19,8 @@
  * files and list directories for use with the gulp live server
  */
 var BBPromise = require('bluebird');
-var app = require('express')();
+var express = require('express');
```

Comment:
I had worked around needing this, so this change is no longer needed.  I'm removing it.